### ISSUE TITLE
Send frame with delay

### ISSUE THREAD

**Issue by atkforever** [miwagner/ESP32-Arduino-CAN#26]
I try to send frame with delay, like this : 

 SendFrame(0x630, 0x2F, 0x00, 0x50, 0x02, 0x01, 0x00, 0x00, 0x00);
 delay(10);
 SendFrame(0x603, 0x2F, 0x00, 0x20, 0x00, 0x00, 0x00, 0x00, 0x00);
 delay(4);
 SendFrame(0x650, 0x23, 0x00, 0x23, 0x04, 0x81, 0x05, 0x9F, 0x37);
 delay(100);
 SendFrame(0x603, 0x2F, 0x01, 0x20, 0x01, 0x32, 0x00, 0x00, 0x00);
  delay(1);
 SendFrame(0x630, 0x2F, 0x00, 0x22, 0x01, 0x01, 0x00, 0x00, 0x00);

So basically 10ms delay, 4, 100 and 1. I have disabled reading in the code so it's only sending. The monitor gives this : 
18:29:30.994 -> Begin
18:29:30.994 -> Frame Sent
18:29:30.994 -> Frame Sent
18:29:30.994 -> Frame Sent
18:29:31.098 -> Frame Sent
18:29:31.098 -> Frame Sent
18:29:31.098 -> End

So, only the 100ms delay seems to work here. Any clue ? My baudrate is 100kps.
